frontend/lib/ui/style/table.css

```
@@ -2,6 +2,7 @@ table.overview_table {
   width: 100%;
   line-height: 1.5rem;
   border-spacing: unset;
+  color: var(--gray-600);
```

```suggestion
  color: var(--color-help-text);
```

gray-600 will be renamed in the design fixes PR